Lib/email/contentmanager.py

```diff
@@ -146,7 +146,8 @@ def embedded_body(lines): return linesep.join(lines) + linesep
     def normal_body(lines): return b'\n'.join(lines) + b'\n'
     if cte==None:
         # Use heuristics to decide on the "best" encoding.
-        if max(len(x) for x in lines) <= policy.max_line_length:
+        if (len(lines) == 0 or
+                max(len(x) for x in lines) <= policy.max_line_length):
```

Comment:
Thanks for the fix! What do you think about using the `default` argument for this:

```python
if max((len(x) for x in lines), default=0) <= policy.max_line_length:
```

?

Reply:
I think using default as you suggest is better. I'll push a new commit.